docs/src/index.md

```diff
@@ -47,7 +50,7 @@ programming.
 ## How the documentation is structured
 
 Having a high-level overview of how this documentation is structured will help
-you know where to look for certain things. There are three main parts:
+you know where to look for certain things. There are three four parts:
```

Comment:
replaced "three main" with "three four" instead of "four main"

Reply:
Oops! Brain fade...